### DISCUSSION TITLE
How to collapse lines with a shortcut?

### DISCUSSION THREAD

**Discussion by volkan0x** [zed-industries/zed#11454] [Feature Requests]
I'd like to know if there is a shortcut to collapse these lines?

<img width="122" alt="image" src="https://github.com/zed-industries/zed/assets/68243739/676fa4c0-a8db-4cf5-982a-1e5e46ddb9de">

**Comment by Moshyfawn** [comment]
It's `alt-cmd-[` for the `editor::Fold` action by default.
